src/routes/IndexRoute.vue

```diff
@@ -1,26 +1,15 @@
 <script lang="ts" setup>
 import DefaultLayout from '../layouts/DefaultLayout.vue';
-import { useI18n } from 'vue-i18n'
+import TrickOverview from '../components/TrickOverview.vue';
+import { ref } from 'vue';
 
-import messages from "../i18n/index"
-
-const i18n = useI18n({
-  messages,
-  useScope: "local"
-})  
-
-const {t} = i18n;
```

Comment:
Why was Internalization removed from the Index-Route?

Feel free to close if it was moved into the child component